src/desktop/apps/artists/index.coffee

```diff
@@ -10,5 +10,5 @@ app = module.exports = express()
 app.set 'views', __dirname + '/templates'
 app.set 'view engine', 'jade'
 
-app.get '/artists', timeout('25s'), routes.index
+# app.get '/artists', timeout('25s'), routes.index
```

Comment:
Might want to add a fixme note here to remove / delete old files once new page is launched